proto/prysm/v1alpha1/validator-client/web_api.proto

```diff
@@ -14,140 +14,175 @@ option java_outer_classname = "WebProto";
 option java_package = "org.ethereum.validator.accounts.v2";
 option php_namespace = "Ethereum\\Validator\\Accounts\\V2";
 
+// DEPRECATED: Prysm Web UI and associated endpoints will be fully removed in a future hard fork.
```

Comment:
Same feedback here about putting the deprecation notice at the end of the comment block to be consistent with the go style of deprecation. I don't feel strongly about this so I won't mark up this file with the code change suggestions.

Reply:
will move this